### ISSUE TITLE
Bike routing over road with foot=no, bicycle=use_sidepath and turn restriction on it.

### ISSUE THREAD

**Issue by hgcvm** [abrensch/brouter#335]
Hello,
Why does BRouter Trekking Bike take a left here: http://brouter.de/brouter-web/#map=17/51.09125/3.44667/standard&lonlats=3.447003,51.090592;3.445115,51.091495

This is the OSM road that shouldn't be used: https://www.openstreetmap.org/way/138475828

There's a turn restriction (but I know that's not considered, except in recumbent bike). But there's also foot=no on it, so bicycle routing shouldn't assume you can dismount there.

So basically: you can't turn left as a bicycle, and you can also not dismount and turn left on foot. 

Mapillary picture of the situation:  https://www.mapillary.com/app/?pKey=1204879349951324

**Comment by afischerdev** [comment]
Hello,

this was discussed in #307. 
The default is changed but the profiles should be changed as well.
When you set 'considerTurnRestrictions' in your sample it will work as expected.

The default is changed but the profiles should be changed as well. I'll do that in next update for #312

**Comment by polyscias** [comment]
Yes, indeed taking the turn-restriction would have helped but the actual problem in my view is that the default trekking profile does not take bicycle=use_sidepath into account, see #79 and #191 

I you look at the details of the route chosen via the "Show detailed route table (T key), you will see that piece of road with bicycle=use_sidepath has a cost of 1450, just like the rest of the Stationssstraat, while I still think bicycle=use_sidepath should be threaded almost the same as bicycle=no.

**Comment by afischerdev** [comment]
Good research, i will add it as well.

**Comment by polyscias** [comment]
Thanks for taking this up!
